libs/api-client/src/api-client.ts

```diff
@@ -14,6 +14,9 @@ const API_GATEWAY =
   'https://gateway.api.dev.globalfishingwatch.org'
 export const USER_TOKEN_STORAGE_KEY = 'GFW_API_USER_TOKEN'
 export const USER_REFRESH_TOKEN_STORAGE_KEY = 'GFW_API_USER_REFRESH_TOKEN'
+export const LAST_API_VERSION = process.env.NEXT_PUBLIC_LAST_API_VERSION || 'v2'
```

Comment:
I'd use `API_VERSION` and `NEXT_PUBLIC_API_VERSION` as variable names, we don't need the `LAST_`